CMakeLists.txt

```diff
@@ -329,30 +326,31 @@ if(CLANG_ADDRESS_SANITIZER)
   set_property(TARGET sb APPEND_STRING PROPERTY COMPILE_FLAGS "-DEXITFREE ")
   set_property(TARGET sb APPEND_STRING PROPERTY COMPILE_FLAGS "-fno-sanitize-recover -fno-omit-frame-pointer -fno-optimize-sibling-calls -fsanitize=address -fsanitize=undefined ")
```

Comment:
Argument `-fno-sanitize-recover` is deprecated, we need to use `-fno-sanitize-recover=undefined,integer`. i think i've changed it for travic CI to work, but we should keep in mind to change this back when travis supports the "new" option.